modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -198,29 +363,43 @@ Some of them can be customized by adding extra configuration to the helm deploym
                                                .1+| `adminService.annotations`     .1+| Annotations object for headless (DNS) service.
```

Comment:
there are no separate fields for standalone and cluster in values.yaml

there is no such key as neo4jService,adminService,externalService

Reply:
replace  the following in standalone 
neo4jService.annotations --> services.default.annotations , 
adminService.annotations --> services.admin.annotations , 
externalService.annotations --> services.neo4j.annotations 

in cluster replace
services.neo4j.annotations  --> services.default.annotations 
annotations --> services.neo4j.annotations